packages/scm/src/browser/scm-commit-widget.tsx

```diff
@@ -142,7 +142,9 @@ export class ScmCommitWidget extends ReactWidget implements StatefulWidget {
                 ref={this.inputRef}
                 rows={1}
                 maxRows={6} /* from VS Code */>
-            </TextareaAutosize>
+            </TextareaAutosize> : '';
```

Comment:
As indicated in the comment above, this ternary isn't necessary, but if you prefer ternary to short-circuit `&&`, `null` or `undefined` would be a more explicit marker of an empty node.

Reply:
Very cool - I like the short-circuit style!